superset-frontend/src/dashboard/containers/DashboardHeader.jsx

```diff
@@ -130,7 +129,6 @@ function mapDispatchToProps(dispatch) {
       dashboardInfoChanged,
       dashboardTitleChanged,
       updateDataMask,
-      fetchUISpecificReport,
```

Comment:
This was not being used anywhere. The only reference to `fetchUISpecificReport` is in `HeaderReportDropdown` which imports the action directly.